examples/magic-wand/magic_wand.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
+Suite Setup                   Setup
+Suite Teardown                Teardown
+Test Setup                    Reset Emulation
+Test Teardown                 Test Teardown
+Resource                      ${RENODEKEYWORDS}
+
+*** Variables ***
+${UART}                       sysbus.uart0
+
+*** Keywords ***
+Create Machine
+    Execute Command          mach create
+    Execute Command          machine LoadPlatformDescription @platforms/boards/arduino_nano_33_ble.repl
+
+    Execute Command          sysbus LoadELF @${CURDIR}/binaries/magic_wand/magic_wand.ino.elf
+
+*** Test Cases ***
+Should Detect RING Motion
+    Create Machine
+    Create Terminal Tester    ${UART}
+
+    Execute Command           sysbus.twi0.lsm9ds1_imu FeedAccelerationSample @${CURDIR}/circle_rotated.data
```

Comment:
I believe we need to download them first, right? And here they are in another dir than in colab

Reply:
oh we clone it anyway